tests/components/Sidebar.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { NavLink } from 'react-router-dom';
+
+import Sidebar from '../../src/components/Sidebar';
+
+describe('Sidebar', () => {
+  it('should have three items', () => {
+    const wrapper = shallow(<Sidebar />);
+    const navList = wrapper.find('.sidebar-list');
+
+    expect(navList.children()).toHaveLength(3);
+  });
+
+  it('should have correct routes on sidebar items', () => {
```

Comment:
I'd prefer to check that when we click on these items, we are actually redirected to the proper component instead of just blindly checking route path.